### PR TITLE
Error in Terminal ,figure out one basic sass error in gulp #121

### PR DESCRIPTION
**Pull request by aayusharora**
There was a typo ,in naming $conf instead of $config.

### CODE REVIEW

File: assets/scss/main.scss [hunk 1]
```diff
@@ -6,73 +6,73 @@
...
-		font-family: map-get($conf, font-family);
+		font-family: map-get($config, font-family);
 	}
 }
 
 $component-toolbar: (
 	theme: map-get($config, theme),
 	name: 'md-toolbar-tools',
-	font-family: map-get($conf, font-family),
+	font-family: map-get($config, font-family),
```
Comment (SaptakS) [inline assets/scss/main.scss]: This change and corresponding changes in other component attributes are enough. You don't need to make change in mixin-component and mixin-text. That is not a typo. It is a parameter which receives the value of `$config`. So, its perfectly fine if it has some other name.

**Comment by aayusharora** [comment]
Thanks @SaptakS ,I have changed the mixin-component and mixin-text.

**Comment by SaptakS** [comment]
@aayusharora Please squash the 2 commits as they are intended to do the same thing. Thanks.

**Comment by aayusharora** [comment]
@SaptakS ,seems correct now?

**Comment by SaptakS** [comment]
@creativepsyco

**Comment by aayusharora** [comment]
@championswimmer , review this please.

**Comment by SaptakS** [comment]
main.css gets generated and injected automatically. I am not really sure you should include that in your commit. @creativepsyco @championswimmer your opinion?

**Comment by championswimmer** [comment]
rebase to top please

File: index.html [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
     <!-- inject:css -->
     <link rel="stylesheet" href="assets/css/leaflet.css">
+    <link rel="stylesheet" href="assets/css/main.css">
```
Comment (championswimmer) [inline index.html]: remove this part. gulp should do this

**Pull request closed without merging**